code/game/machinery/vending/cm_vending.dm

```diff
@@ -465,6 +481,16 @@ IN_USE						used for vending/denying
 	update_icon()
 	return
 
+//-----------TGUI PROCS------------------------
+/obj/structure/machinery/cm_vending/ui_static_data(mob/user)
+	. = list()
```

Comment:
`.`-hijacking is generally discouraged, would be better to do var/list/data = list() and then return data

Reply:
Gotcha.